installation.md

```diff
@@ -69,45 +69,47 @@ should work with any of them, though feature set and interface may vary.
 Here are some plugins we know work well with clangd:
 
 <details>
-<summary markdown="span">Vim</summary>
-[YouCompleteMe](https://valloric.github.io/YouCompleteMe/) can be installed with
-clangd support. **This is not on by default**, you must install it with
-`install.py --clangd-completer`.
+<summary markdown="span">Vim/Neovim</summary>
+We recommend to use [coc.nvim](https://github.com/neoclide/coc.nvim) + [coc-clangd](https://github.com/clangd/coc-clangd)
+in Vim/Neovim.
 
-We recommend changing a couple of YCM's default settings. In `.vimrc` add:
-```
-" Let clangd fully control code completion
-let g:ycm_clangd_uses_ycmd_caching = 0
-" Use installed clangd, not YCM-bundled clangd which doesn't get updates.
-let g:ycm_clangd_binary_path = exepath("clangd")
-```
+1. install [Node.js](https://nodejs.org/), both `coc.nvim` and `coc-clangd` run on Node.js.
+2. install `coc.nvim` by [vim-plug](https://github.com/junegunn/vim-plug)
+(check out [coc.nvim Wiki](https://github.com/neoclide/coc.nvim/wiki/Install-coc.nvim) for other options):
+   - add `Plug 'neoclide/coc.nvim', {'branch': 'release'}` to `.vimrc/init.vim`
+   - run `:PlugInstall` in Vim
+3. in Vim, run `:CocInstall coc-clangd`
+4. `coc-clangd` will try to find clangd from your $PATH, if not found, you can
```

Comment:
I'd suggest just "If you haven't installed clangd yet" rather than get into the mechanics